@Test public void should_fail_if_atomicIntegerFieldUpdater_does_not_contain_expected_value(){
  AtomicIntegerFieldUpdater<Person> fieldUpdater=AtomicIntegerFieldUpdater.newUpdater(Person.class,"age");
  assertThatExceptionOfType(AssertionError.class).isThrownBy(() -> assertThat(fieldUpdater).hasValue(25,person)).withMessage(shouldHaveValue(fieldUpdater,person.age,25,person).create());
}
